packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -92,6 +93,7 @@ type MafsGraphProps = Props & {
 export const MafsGraph = (props: MafsGraphProps) => {
     const {state, dispatch} = props;
     const [width, height] = props.box;
+    const {labels} = useGraphConfig();
```

Comment:
I originally had labels in the props, but since labels can be found in the context, I switched to this. Since the jsx returned by this component is what contains the reference to the context though, can I do this?

Reply:
Nope. Looks like that's a no XD